[new] using read-only replica (#1810)
timescale/docs: mst/postgresql-read-replica.md

the replica and reduce the load on the primary server. You can optimize query
response times across different geographical locations because the replica can
be created in different regions or on different cloud providers.

> For information about creating a read-only replica using aiven client,

Reviewer: ```suggestion
For information about creating a read-only replica using the Aiven client,
```

project. The `Overview` tab of the replica also lists the name of the primary
service for the replica. To promote a read-only replica as a master database,
click the `Promote to master` button.

<procedure>

## Using read-only replica for the service on MST

1.  In the `Overview` page of the read-only replica for the service on MST, copy
    the `Service URI`.

> 1.  Connect to the read-only service using psql client:

Reviewer: ```suggestion
1.  At the psql prompt, connect to the read-only service:
```

    ```sql

>        psql <SERVICE_URI>

Reviewer: Line the command up with the three backticks

Author: done

    ```

1.  To check whether you are connected to a primary or replica node:

    ```sql

>      SELECT * FROM pg_is_in_recovery();

Reviewer: Same here

    ```

> If the output is `TRUE` you are connected to the replica, and if the output is

Reviewer: Line this up with the previous para ("To check whether ...")

> `FALSE` if you are connected to the primary server.

Reviewer: ```suggestion
`FALSE` you are connected to the primary server.
```

</procedure>

<highlight type="note">

> Managed Service for TimescaleDB uses asynchronous replication and hence a lag is

Reviewer: ```suggestion
Managed Service for TimescaleDB uses asynchronous replication, so some lag is
```

> expected. When running an `INSERT` operation on the primary node, a minimal

Reviewer: ```suggestion
expected. When you run an `INSERT` operation on the primary node, a small
```
